src/libpmemfile-posix/file.c

```diff
@@ -359,6 +359,20 @@ _pmemfile_openat(PMEMfilepool *pfp, struct pmemfile_vinode *dir,
 		goto end;
 	}
 
+	if (flags & PMEMFILE_O_PATH)
```

Comment:
So these dozen lines of code is moved from the end of the function to here. Why, by that way?